api-microservice/src/services/Logger.ts

```diff
@@ -0,0 +1,23 @@
+import * as winston from 'winston';
+
+export default class LoggerService {
+  private logger: winston.Logger;
+
+  public constructor() {
+    const loggerSettings = {
+      format: winston.format.combine(winston.format.timestamp(), winston.format.json()),
+      level: 'info',
+      transports: [
+        new winston.transports.Console(),
+        new winston.transports.File({ filename: 'error.log', level: 'error' }),
```

Comment:
Can we just support console logging? Adding it into logs isn't necessary for Docker (and can cause problems)

We might need to revisit the format in future - I'm not 100% how it plays with Azure logging

Reply:
yep, no problem, wasn't 100% we needed file logging

Reply:
format config is easily maintainable so yes, can change it anytime.